src/python/grpcio/README.rst

```diff
@@ -51,7 +51,7 @@ package named :code:`python-dev`).
   $ git submodule update --init
 
   # For the next two commands do `sudo pip install` if you get permission-denied errors
-  $ pip install -rrequirements.txt
+  $ pip install -requirements.txt
```

Comment:
Thats actually correct.

Reply:
yes. I see. So I changed to `pip install -r requirements.txt`